entrypoint.sh

```diff
@@ -1,9 +1,18 @@
 #!/bin/bash
 
+export TARGET_API="https://api.qa.notprod.lev.homeoffice.gov.uk"
+export PATH="$PATH:./node_modules/.bin"
```

Comment:
I'm not sure this wants `export` does it?

The scope should be limited to this script.